presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -66,6 +66,10 @@ Using ``NULL`` with ``IS NULL`` evaluates to true::
 But any other constant does not::
 
     SELECT 3.0 IS NULL; -- false
+    
+Note that ``nan()`` is different to ``NULL``, and counts as ``NOT NULL``::
```

Comment:
nit for all of these: "different from" is more standard than "different to" (especially in American English).

```suggestion
Note that ``nan()`` is different from ``NULL``, and is considered ``NOT NULL``::
```

Reply:
I like this! For consistency, could we consider using this suggested phrasing for all of these? 

```suggestion
Note that ``nan()`` is different from ``NULL``, and is considered ``NOT NULL``::
```